### ISSUE TITLE
AVAnnotate manifests don't show media resources

### ISSUE THREAD

**Issue by Dananji** [samvera-labs/ramp#710]
### Description

AVAnnotate manifest from the interoperability testing worksheet doesn't load the media into the player.

Links to the manifests:
https://tanyaclement.github.io/znh-1939/manifests/manifest.json
https://avannotate.github.io/SuspenseTheLonelyVilla/manifests/manifest.json

Both these manifest display next/previous buttons, but no message. These don't seem to be restricted items, as they are loading and playing in other viewers.

### Done Looks Like
- [x] Load the media resource from the manifest

**Comment by elynema** [comment]
I did hear there may be some issues with the AVAnnotate manifests themselves, so should run these through the IIIF validator before developing around them.

**Comment by saracarl** [comment]
Yeah, the IIIF validator hangs because it can't handle "range" stanzas.  There's an issue open on the validator for that.

**Comment by joncameron** [comment]
👍
